tarantula.lic

```diff
@@ -7,21 +7,24 @@ custom_require.call(%w(common drinfomon))
 no_kill_all
 
 class Tarantula
-  include DRC
 
   def initialize
     settings = get_settings
     @tarantula = settings.tarantula
-    @startup_delay = settings.tarantula_startup_delay || 15
-    @tarantula_noun = settings.tarantula_noun.nil? ? 'biomechanical tarantula' : settings.tarantula_noun
+    @startup_delay = settings.tarantula_startup_delay
+    @tarantula_noun = settings.tarantula_noun
+    unless DRCI.exists?(@tarantula_noun)
```

Comment:
`common-items` needs to be added to `custom_require` near the top

Reply:
Do you use an IDE that quickly points that out to you, or do you just the The Eye for that kind of thing? I'm using vim for these, and seriously considering going RubyMine, or VSCode if they'll help me spot this kind of thing easier.

Reply:
> Do you use an IDE that quickly points that out to you, or do you just the The Eye for that kind of thing? I'm using vim for these, and seriously considering going RubyMine, or VSCode if they'll help me spot this kind of thing easier.

I use VSCode and it doesn't help me with this, but maybe I don't have it set up right.

I think @rpherbig is just damn good at this. Hah.

Reply:
I don't use an IDE, just GitHub's diff interface